html/donate/hi.html

```diff
@@ -3,7 +3,7 @@
   <head>
     <meta charset="UTF-8" />
 
-    <title>Donate to Open Food Facts</title>
+    <title>ओपन फूड फैक्ट्स को दान करें</title>
```

Comment:
I don't think it makes sense to translate "Open Food Facts". (same for other strings in Hindi below)